packages/instrumentation-socket.io/src/socket.io.ts

```diff
@@ -206,6 +290,9 @@ export class SocketIoInstrumentation extends InstrumentationBase<Io> {
                 if (!self._config.traceReserved && reservedEvents.includes(ev)) {
                     return original.apply(this, arguments);
                 }
+                if (Array.isArray(self._config.emitIgnoreEventList) && self._config.emitIgnoreEventList.includes(ev)) {
```

Comment:
This feature is loosely tested for v2, but it could be nice to also add a dedicated test that will run for all versions and will verify for non-reserved events